src/meghaJJune20/Bank.java

```diff
@@ -0,0 +1,77 @@
+package meghaJJune20;
+
+public class Bank {
+	
+	int countdebit;
+	int countcredit;
+	int countbalance;
+	int userid;
+	
+	static int debittotal=0;
+	static int credittotal=0;
+	static int baltotal=0;
+		
+	void setData( int userid1,int balance)
+	{
+		userid=userid1;
+		countbalance=balance;
+	}
+	void debitamount(int debit1)
+	{
+		countbalance=countbalance-debit1;
+		countdebit++;
+		debittotal++;
+	}
+	void creditamount(int credit1)
+	{
+		countbalance=countbalance+credit1;
+		countcredit++;
+		credittotal++;	
+	}
+	void printbalance()
+	{
+		System.out.println();
+		System.out.println("Balance of User " +userid+" is "+countbalance);
+		countbalance++;
+		baltotal++;
+	}
+	void individualTransactionSummary()
+	{
+		System.out.println();
+		if(userid==1)
```

Comment:
No Need to write if block here. Because you already created instance variable for the userID.

Reply:
Done as per comments